BEP3.md

```diff
@@ -80,6 +80,15 @@ APS is designed to support peg token from any EVM based blockchain or any one wi
 3. Client or Client tooling monitors events of the APS contract on Ethereum. If it detects Deputy process’s call, it will verify and call the APS contract to claim the tokens by disclosing the random number generating the hash.
 4. Deputy process monitors the events from APS contract. If there is a successful claim, it will read the value of the random number Client disclosed and broadcast a CHLT transaction to claim the locked tokens by the Client via the HTLT.
 
+### Swap between several BEP2 tokens
+
+Suppose a client has some `BNB` token and want to swap some other BEP2 tokens, like `ABC` and `DEF`:
```

Comment:
please highlight 2 motivations for this:

1. users want to swap assets that are not listed on Binance DEX;
2. users want to swap one assets with several others at the same time.